SConstruct

```diff
@@ -118,6 +118,13 @@ external = env.Command([Dir(os.path.join(REPO_ROOT, "HSC", "masks")),
                                          REPO_ROOT, "gaia_dr2_20200414", "refcats",
                                          os.path.join(TESTDATA_ROOT, "gaia_dr2_20200414.ecsv")),
 
+                        getExecutableCmd("daf_butler", "butler", "register-dataset-type", REPO_ROOT,
+                                         "gaia_dr3_20230707", "SimpleCatalog", "htm7"),
+                        getExecutableCmd("daf_butler", "butler", "ingest-files",
```

Comment:
Why not just replace the dr2 versions of these lines above? Is the intent to have both of them in ci_hsc for a while? I would have thought we could just swap out dr2 for dr3.

Reply:
Ok, removing DR2 is blocked because it is still used in `analysis_tools` (which I knew, but forgot would affect `ci_hsc_gen3`. I had originally planned to switch to DR3 in `analysis_tools` too as part of this ticket, but there is a refactoring of the relevant code going on right now, so I don't want to mess with it.

Reply:
Ok, please leave a comment to that effect here, so others understand why both are there. Maybe also link to the relevant "DR2 isn't used in analysis_tools" ticket, too?